cross/openldap/PLIST

```diff
@@ -0,0 +1,27 @@
+bin:bin/ldapadd
+bin:bin/ldapcompare
+bin:bin/ldapdelete
+bin:bin/ldapexop
+bin:bin/ldapmodify
+bin:bin/ldapmodrdn
+bin:bin/ldappasswd
+bin:bin/ldapsearch
+bin:bin/ldapurl
+bin:bin/ldapwhoami
+etc:etc/openldap/ldap.conf
+etc:etc/openldap/ldap.conf.default
+lnk:lib/liblber-2.4.so.2
+lib:lib/liblber-2.4.so.2.10.3
+lib:lib/liblber.a
+rsc:lib/liblber.la
+lnk:lib/liblber.so
+lnk:lib/libldap-2.4.so.2
+lib:lib/libldap-2.4.so.2.10.3
+lib:lib/libldap.a
+rsc:lib/libldap.la
+lnk:lib/libldap_r-2.4.so.2
+lib:lib/libldap_r-2.4.so.2.10.3
+lib:lib/libldap_r.a
+rsc:lib/libldap_r.la
+lib:lib/libldap_r.so
+lib:lib/libldap.so
```

Comment:
Remove all *.a and *.la files, we don't need that on the target. Also maybe etc/ stuff.